<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>20130778 - Transformaciones 3D</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/favicon.png</normaloff>:/img/favicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>241</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);</string>
    </property>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>181</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249, 249, 249);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(95, 95, 95);
border-style: solid;
border-width: 0px;
border-color: none;</string>
      </property>
      <property name="text">
       <string>Animaciones</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDetenerA3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(100, 100, 100);
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDetenerA2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(100, 100, 100);
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDetenerA1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(100, 100, 100);
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAnimacion1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(100, 100, 100);
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
      </property>
      <property name="text">
       <string>Animación 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAnimacion2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(100, 100, 100);
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
      </property>
      <property name="text">
       <string>Animación 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAnimacion3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(100, 100, 100);
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
      </property>
      <property name="text">
       <string>Animación 3</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>181</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249, 249, 249);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>131</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(95, 95, 95);
border-style: solid;
border-width: 0px;
border-color: none;</string>
      </property>
      <property name="text">
       <string>Colores</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMasColores">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Seleccionar color</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(95, 95, 95);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnArcoiris">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="img.qrc">
        <normaloff>:/img/arcoiris.png</normaloff>:/img/arcoiris.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QSlider" name="vsIntensidadR">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>16</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: none;
border-radius: none;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="sliderPosition">
       <number>180</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="vsIntensidadB">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>16</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: none;
border-radius: none;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="sliderPosition">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="vsIntensidadG">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>16</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: none;
border-radius: none;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="sliderPosition">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDetenerArcoiris">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>83</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(100, 100, 100);
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
      </property>
      <property name="text">
       <string>Detener</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249, 249, 249);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(95, 95, 95);
border-style: solid;
border-width: 0px;
border-color: none;</string>
      </property>
      <property name="text">
       <string>Grosor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDial" name="dialGrosor">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249, 249, 249);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QDial" name="dialAlpha">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>45</number>
      </property>
      <property name="notchesVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(95, 95, 95);
border-style: solid;
border-width: 0px;
border-color: none;</string>
      </property>
      <property name="text">
       <string>Alpha</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDial" name="dialPhi">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>61</width>
        <height>64</height>
       </rect>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
      <property name="notchesVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(95, 95, 95);
border-style: solid;
border-width: 0px;
border-color: none;</string>
      </property>
      <property name="text">
       <string>Phi</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>600</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);</string>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(95, 95, 95);
border-style: solid;
border-width: 0px;
border-color: none;</string>
     </property>
     <property name="text">
      <string>Usar teclado</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReset">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>6</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(100, 100, 100);
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: rgb(234, 234, 234);
border-radius: 8px;</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
